docs/features/blocks/breadcrumbs.md

```diff
@@ -0,0 +1,150 @@
+---
+id: breadcrumbs
+title: "Blocks - Breadcrumbs"
+sidebar_label: Breadcrumbs
+description: This documentation provides information about breadcrumbs block.
+---
+
+This documentation provides information about the breadcrumb block
+that [Yoast SEO](https://yoast.com/wordpress/plugins/seo/) provides.
+
+## Filters
+
+### Changing the output {#breadcrumb_output}
+
+The `wpseo_breadcrumb_output` filter takes the output of the breadcrumb. Make sure not to empty the output, just replace
+or add data. An example:
+
+```php
+add_filter( 'wpseo_breadcrumb_output', [ $this, 'override_home_icon' ] );
```

Comment:
Here and below too, using `$this` in each example makes it only usable inside classes. Wouldn't it be easier to try if we provide simple file examples? So users can just paste the example in the theme's `functions.php`?

E.g. 
```suggestion
add_filter( 'wpseo_breadcrumb_output', 'override_home_icon' );
```

Plus change below to not be `public`